### PR TITLE
pings: bootstrap initial code logic

### PR DESCRIPTION
**Pull request by unknwon**
Bootstraps the new Pings service that currently replicates whatever we have today for the dotcom Ping endpoint (`https://sourcegraph.com/.api/updates`).

## Test plan

Please see https://docs.google.com/document/d/1knDauzIh0cW52RUOFTRQ3SHttTepcGUo69zmr4ZhSEw/edit#heading=h.n1e4w4vwvdto

---

Part of https://github.com/sourcegraph/sourcegraph/issues/55597

### CODE REVIEW

**Review by rafax — APPROVED** [review]
Looks good to me on a first glance, I’m OOO tomorrow so I can try to run this on Wednesday at the earliest.

Approving to unblock (looks like boilerplate / bootstrapping, so I hope it's not harmful)

File: dev/check/go-dbconn-import.sh [hunk 1]
```diff
@@ -27,6 +27,8 @@ allowed_prefix=(
   github.com/sourcegraph/sourcegraph/cmd/symbols
   # Transitively depends on zoekt package which imports but does not use DB
   github.com/sourcegraph/sourcegraph/cmd/searcher
+  # Transitively depends on updatecheck package which imports but does not use DB
+  github.com/sourcegraph/sourcegraph/cmd/ping-service
```
Comment (bobheadxi) [inline dev/check/go-dbconn-import.sh:31]: are you planning on trying to remove this transient dependency?

**Review by bobheadxi — APPROVED** [review]
LGTM, minor nit would be `cmd/ping-service` seems unnecessarily verbose when we could just use `cmd/pings`

File: dev/check/go-dbconn-import.sh (hunk 1 above)
Comment (unknwon) [reply dev/check/go-dbconn-import.sh:31]: Not actually planned unless we want to do some package splitting/refactoring for `internal/updatecheck`.

**Review by bobheadxi — APPROVED** [review]
✨

**Comment by unknwon** [comment]
Merging as-is since it's a net new service that isn't deployed anywhere and affects no one, happy to address post-merge feedback!

**Pull request merged**